controllers/lmes/lmevaljob_controller.go

```diff
@@ -634,6 +643,41 @@ func (r *LMEvalJobReconciler) handleCancel(ctx context.Context, log logr.Logger,
 	return ctrl.Result{}, err
 }
 
+func (r *LMEvalJobReconciler) handleSuspend(ctx context.Context, log logr.Logger, job *lmesv1alpha1.LMEvalJob) (ctrl.Result, error) {
+	defer r.pullingJobs.remove(string(job.GetUID()))
+	if job.Status.State != lmesv1alpha1.NewJobState {
+		log.Info("Suspend job")
+		if err := r.deleteJobPod(ctx, job); err != nil && client.IgnoreNotFound(err) != nil {
+			log.Error(err, "failed to delete pod for suspended job")
+			return ctrl.Result{Requeue: true, RequeueAfter: r.options.PodCheckingInterval}, err
```

Comment:
please update this to `return r.pullingJobs.addOrUpdate(string(job.GetUID()), r.options.PodCheckingInterval), nil` since the pulling mechanism is merged.